src/github.com/getlantern/flashlight/pac.go

```diff
@@ -9,11 +10,13 @@ import (
 	"github.com/getlantern/filepersist"
 	"github.com/getlantern/pac"
 
-	"github.com/getlantern/flashlight/proxiedsites"
+	"github.com/getlantern/flashlight/ui"
```

Comment:
This is a strange dependency.  I like that you made detour its own library, which makes it useful to other people.  However, if it has dependencies on flashlight, it becomes less useful.  I would invert the dependency so that the caller of detour passes it a callback (or something similar) which detour can use to update the PAC file stuff.